juniper_codegen/src/util.rs

```diff
@@ -593,6 +593,11 @@ pub struct GraphQLTypeDefinitionField {
     pub resolver_code: proc_macro2::TokenStream,
 }
 
+pub fn unraw(s: &str) -> String {
```

Comment:
I don't think we really need this helper.

`arg.name.unraw().to_string()` would be the same.

Reply:
[`unraw`](https://docs.rs/syn/1.0.7/syn/ext/trait.IdentExt.html) only works on `Ident` and the names here are `String`s. So it would become `format_ident!("{}", field.name).unraw().to_string();` which is a bit wordy. I don't mind changing it if you still think the helper is unnecessary.

Reply:
Ah, I missed that it was taking a `&str`. I think we will want some general refactoring of the macro code to keep the `Ident`s around to provide better error messages.

But we can do that in future work.